tox.ini

```diff
@@ -0,0 +1,150 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, unit, static, coverage-report
+
+[vars]
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
+plugins_path = {toxinidir}/indico_rock/plugins/
```

Comment:
Can be removed, it's a leftover from indico

Reply:
Oops, good catch, removed!